funds-manager/funds-manager-server/src/execution_client/quotes.rs

```diff
@@ -19,6 +20,9 @@ impl ExecutionClient {
         let url = format!("{}?{}", QUOTE_ENDPOINT, query_string);
 
         let resp: LiFiQuote = self.send_get_request(&url).await?;
+
+        info!("Got LiFi quote from: {}", resp.tool);
```

Comment:
is it possible to also log the tool alongside the swap details, maybe with the "Swap executed at ..." as metadata?